### ISSUE TITLE
Typescript blocks access to pagination result "start" and "next" properties

### ISSUE THREAD

**Issue by mobsense** [sensedeep/dynamodb-onetable#19]
Model.find returns a `start` and `next` properties for pagination. The Typescript Model.find signature blocks access to these.

Also OneParams.start should be objec.

**Comment by mobsense** [comment]
Case:

```
const result: UserMessage[] = await UserMessageModel.find({ userId }, { limit, start })
```

UserMessage[] will not have a start property

**Comment by mobsense** [comment]
Workaround:

```
const result: UserMessage[] = await UserMessageModel.find({ userId }, { limit, start })
interface Result {
    start?: string
}
const { start } = result as Result
```

**Comment by mobsense** [comment]
Fixed in 1.3.0
